blockchain/chain/src/store/chain_store.rs

```diff
@@ -98,6 +99,12 @@ pub struct ChainStore<DB> {
 
     /// Tracks blocks for the purpose of forming tipsets.
     tipset_tracker: TipsetTracker<DB>,
+
+    /// File backed genesis block header
+    file_backed_genesis: Mutex<FileBacked<BlockHeader>>,
```

Comment:
Let's just store the genesis CID. It is a lot smaller than the entire genesis block.

Reply:
Fixed